.ci/appveyor.yml

```diff
@@ -56,6 +56,15 @@ install:
   - "npm config set loglevel warn"
   - "npm install"
 
+  - curl -fsSL ftp://ftp.zlatkovic.com/libxml/iconv-1.9.2.win32.zip -o iconv.zip
```

Comment:
@adtac It is installing the packages correctly, `curl` and `7z` must be installed by default I guess